L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -447,15 +447,15 @@ namespace trklet {
     //have the factor if 2
     double krprojshiftdisk() const { return 2 * kr(); }
 
-    double benddecode(unsigned int ibend, unsigned int layerdisk, bool isPSmodule) const {
+    double benddecode(int ibend, int layerdisk, bool isPSmodule) const {
       if (layerdisk >= N_LAYER && (!isPSmodule))
-        layerdisk += N_DISK;
+        layerdisk += (N_LAYER - 1);
```

Comment:
Why this change? Was original line a bug?

Reply:
I'm not sure how this got changed. I'll change this back.